downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

```diff
@@ -74,8 +90,18 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 
 ### New features
 
+#### Persist Tool
+
+- Added support for the `In-Memory` data store. Set as the default data store.
+- Added support for the `Google Sheets` data store. The support for Google Sheets data store is currently an experimental feature, and its behavior may be subject to change in future releases.
+- Added `persist migrate` command to migration scripts for data model changes in the `mysql` data store. The support for migrations is currently an experimental feature, and its behavior may be subject to change in future releases.
+
 ### Improvements
 
+#### Persist Tool
+- Added support for additional DB configurations in MySQL generated client objects.
```

Comment:
```suggestion
- Added support for additional DB configurations in MySQL generated client objects.
```
Should it be `MySQL-generated`?

Reply:
Better it change to 'generated MySQL client object. WDYT?